CHANGELOG.md

```diff
@@ -2,11 +2,14 @@
 
 ## [Unreleased]
 
+* Removed plugin
+  * Removed `stylelint-order` because it adds unnecessary overhead to writing CSS
```

Comment:
True, I did show my bias here a bit.  Happy to update before/if this merges based on the consensus.